tox.ini

```diff
@@ -13,9 +13,12 @@ python =
 setenv =
     PATH={toxinidir}/node_modules/.bin{:}{envbindir}{:}{env:PATH}
 deps = -rrequirements_dev.txt
+allowlist_externals =
```

Comment:
`allowlist_externals` is what replaced `whitelist_externals` for recent versions of tox.